claim/tests.py

```diff
@@ -1 +1,110 @@
+        cls.admin_token = get_token(cls.admin_user, DummyContext(user=cls.admin_user))
+    
+    def test_claims_query(self):
+        
+        response = self.query(
+            '''
+            query {
+                claims
+                {
+                    totalCount
+                    pageInfo { hasNextPage, hasPreviousPage, startCursor, endCursor}
+                    edges
+                    {
+                        node
+                        {
+                            uuid,code,jsonExt,dateClaimed,dateProcessed,feedbackStatus,reviewStatus,claimed,approved,status,restore {id},healthFacility { id uuid name code },insuree{id, uuid, chfId, lastName, otherNames, dob},attachmentsCount
+                        }
+                    }
+                }
+            }
+            ''',
+            headers={"HTTP_AUTHORIZATION": f"Bearer {self.admin_token}"},
+        )
+
+        content = json.loads(response.content)
+
+        # This validates the status code and if you get errors
+        self.assertResponseNoErrors(response)
+
+        # Add some more asserts if you like
+        ...
+
+    def test_query_with_variables(self):
+        response = self.query(
+            '''
+    
+            query claims($status: Int!, $first:  Int! ) {
+                claims(status: $status,orderBy: ["-dateClaimed"],first: $first)
+                {
+                    totalCount
+                    pageInfo { hasNextPage, hasPreviousPage, startCursor, endCursor}
+                    edges
+                    {
+                        node
+                        {
+                            uuid,code,jsonExt,dateClaimed,dateProcessed,feedbackStatus,reviewStatus,claimed,approved,status,restore {id},healthFacility { id uuid name code },insuree{id, uuid, chfId, lastName, otherNames, dob},attachmentsCount
+                        }
+                    }
+                }
+            }
+            ''',
+            headers={"HTTP_AUTHORIZATION": f"Bearer {self.admin_token}"},
+            variables={'status': 2, 'first':10}
+        )
+
+        content = json.loads(response.content)
+
+        # This validates the status code and if you get errors
+        self.assertResponseNoErrors(response)
+
+        # Add some more asserts if you like
+
+    #def test_some_mutation(self):
+    #    response = self.query(
+    #        '''
+    #        mutation myMutation($input: MyMutationInput!) {
+    #            myMutation(input: $input) {
+    #                my-model {
+    #
+    #                    id
+    #                    name
+    #                }
+    #            }
+    #        }
+    #        ''',
+    #        op_name='myMutation',
+    #        input_data={'my_field': 'foo', 'other_field': 'bar'}
+    #    )
+
+        # This validates the status code and if you get errors
+    #    self.assertResponseNoErrors(response)
+
+        # Add some more asserts if you like
```

Comment:
I think this section can be removed

Reply:
I wanted to keep it as an example to add few mutations test but we can remove it and put it back once it will be real test